lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -73,9 +76,11 @@ class NetworkAnalyzer {
     return summaryByKey;
   }
 
+  /** @typedef {{record: LH.Artifacts.NetworkRequest, timing: LH.Crdp.Network.ResourceTiming, connectionReused?: boolean}} RequestInfo */
```

Comment:
I apologize for the terrible name here, @patrickhulce :)

Reply:
heh, works for me :)